recipes/server.rb

```diff
@@ -89,7 +89,7 @@
 
 windows_package package_name do
   source !is_iso ? package_url : "#{iso_extraction_dir}/#{node['sql_server']['server']['setup']}"
-  checksum package_checksum
+  checksum !is_iso ? package_checksum : nil
```

Comment:
An explanation: We just tell windows_package not to check, because the checksum is already verified on the iso file [during downloading](https://github.com/Aqovia/sql_server/blob/b898294b44efc8bc9500c0bc49b6d7673d508ead/recipes/server.rb#L75).